<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="gif_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>560</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sound.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>950</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Siguiente</string>
    </property>
   </widget>
   <widget class="QToolButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>950</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Escuchar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>521</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 55pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>   Escucha 
atentamente</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
